acceptance/.shared/kitchen_acceptance/libraries/kitchen.rb

```diff
@@ -31,6 +31,7 @@ class Kitchen < Chef::Resource
     property :artifactory_password, String, default: lazy { ENV["ARTIFACTORY_PASSWORD"] ? ENV["ARTIFACTORY_PASSWORD"] : "" }
     property :env, Hash, default: {}
     property :kitchen_options, String, default: lazy { ENV["PROJECT_NAME"] ? "-c -l debug" : "-c" }
+    property :kitchen_log_path, String, default: lazy { ENV["WORKSPACE"] ? "#{ENV["WORKSPACE"]}/chef-acceptance-data/logs" : "#{kitchen_dir}/../.acceptance_data/logs/kitchen" }
```

Comment:
I'd just reference this as a local variable in the ruby block instead of exposing as a property on the resource